packages/zcli-apps/src/commands/apps/new.ts

```diff
@@ -41,7 +51,8 @@ export default class New extends Command {
       destination.on('finish', () => {
         const zip = new AdmZip(this.zipScaffoldPath)
         const overwrite = false
-        zip.extractAllToAsync(path.join(process.cwd()), overwrite, (err) => {
+        zip.extractAllToAsync(path.join(process.cwd()), overwrite, async (err) => {
+          await tryCleanUp(this.zipScaffoldPath)
```

Comment:
not sure `await tryCleanUp(this.zipScaffoldPath)` will work here given tryCleanUp is not a promise. Actually, it might ok to just use `await cleanDirectory(path)` directly here instead and let error bubble up.